GooglePlacesAutocomplete.js

```diff
@@ -152,6 +168,11 @@ export default class GooglePlacesAutocomplete extends Component {
     if (prevProps.predefinedPlaces !== this.props.predefinedPlaces) {
       this.setState({ dataSource: this.buildRowsFromResults(this._results) })
     }
+    if (!('searchNudgesDecorator' in this.state.dataSource[0])) {
```

Comment:
probably want to make sure that `props.searchNudgesQuery` is also defined
```suggestion
    if (this.props.searchNudgesQuery && !('searchNudgesDecorator' in this.state.dataSource[0])) {
```